services/movie.py

```diff
@@ -0,0 +1,43 @@
+from django.db.models import Q, QuerySet
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None) -> QuerySet:
+    query = Movie.objects.all()
+    if genres_ids and actors_ids:
+        query = query.filter(
+            Q(genres__id__in=genres_ids)
+            & Q(actors__id__in=actors_ids))
+    elif genres_ids:
+        query = query.filter(genres__id__in=genres_ids)
+    elif actors_ids:
+        query = query.filter(actors__id__in=actors_ids)
+    return query.distinct()
+
+
+def get_movie_by_id(move_id: int) -> Movie:
+    return Movie.objects.get(id=move_id)
+
+
+def create_movie(
+        movie_title: str = None,
+        movie_description: str = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
+) -> None:

```

Comment:
The parameters `movie_title`, `movie_description`, `genres_ids`, and `actors_ids` have default values of `None`. This could lead to issues if the function is called without arguments. Consider adding validation to ensure these parameters are provided and are valid (e.g., non-empty string for `movie_title`, valid lists for `genres_ids` and `actors_ids`).